crates/diesel_models/src/query/user/sample_data.rs

```diff
@@ -165,3 +182,30 @@ pub async fn delete_refunds(
             _ => Ok(result),
         })
 }
+
+pub async fn delete_disputes(
+    conn: &PgPooledConn,
+    merchant_id: &common_utils::id_type::MerchantId,
+) -> StorageResult<Vec<Dispute>> {
+    // delete disputes where merchant_id = merchant_id and payment_id like "test_%"
+    let query = diesel::delete(<Dispute>::table())
+        .filter(dispute_dsl::merchant_id.eq(merchant_id.to_owned()))
+        .filter(dispute_dsl::payment_id.like("test_%"));
```

Comment:
Payment_id `test_` is fine, but shouldn't we use dispute id like `test_`, that will be more apt.